cobbler/actions/buildiso.py

```diff
@@ -167,7 +167,7 @@ def generate_netboot_iso(self, imagesdir, isolinuxdir, profiles=None, systems=No
             data = utils.blender(self.api, False, profile)
 
             # SUSE is not using 'text'. Instead 'textmode' is used as kernel option.
-            kopts_overwrite(system, dist, data['kernel_options'], self.api.settings())
+            utils.kopts_overwrite(system, dist, data['kernel_options'], self.api.settings())
```

Comment:
@watologo1 But it seems like there is still a problem with `system`. Where should this be coming from? Do we need it?

Reply:
There is another caller in tftpgen.py.
Everything should be fine now. Thanks!